internal/service/projectipaccesslist/resource_project_ip_access_list.go

```diff
@@ -69,6 +69,7 @@ func (r *projectIPAccessListRS) Create(ctx context.Context, req resource.CreateR
 		Target:  []string{"created", "failed"},
 		Refresh: func() (any, string, error) {
 			_, _, err := connV2.ProjectIPAccessListApi.CreateProjectIpAccessList(ctx, projectID, NewMongoDBProjectIPAccessList(projectIPAccessListModel)).Execute()
+			// Atlas Create is called inside refresh because this limitation: This endpoint doesn't support concurrent POST requests. You must submit multiple POST requests synchronously.
```

Comment:
[nitpick] Consider rewording this comment to explicitly state the Atlas API's limitation on handling concurrent POST requests and to clarify why a synchronous approach is required.

```suggestion
			// Atlas Create endpoint doesn't support concurrent POST requests. Code submits multiple POST requests synchronously.
```

Reply:
thanks, I want to cite verbatim the Atlas doc in https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Project-IP-Access-List/operation/createProjectIpAccessList